src/test/scala/uk/gov/nationalarchives/tdr/api/service/FFIDMetadataServiceSpec.scala

```diff
@@ -81,6 +83,82 @@ class FFIDMetadataServiceSpec extends AnyFlatSpec with MockitoSugar with Matcher
     matches.puid.get shouldEqual "puid"
   }
 
+  "getFFIDMetadata" should "call the repository with the correct arguments" in {
+    val ffidMetadataRepositoryMock = mock[FFIDMetadataRepository]
+    val consignmentIdCaptor: ArgumentCaptor[UUID] = ArgumentCaptor.forClass(classOf[UUID])
+    val consignmentId = UUID.randomUUID()
+
+    when(ffidMetadataRepositoryMock.getFFIDMetadata(consignmentIdCaptor.capture())).thenReturn(Future(Seq()))
+
+    val service = new FFIDMetadataService(ffidMetadataRepositoryMock, mock[FFIDMetadataMatchesRepository], FixedTimeSource, new FixedUUIDSource())
+    service.getFFIDMetadata(consignmentId)
+
+    consignmentIdCaptor.getValue should equal(consignmentId)
+  }
+
+  "getFFIDMetadata" should "return the correct values" in {
```

Comment:
Could this test name be more specific than "correct values"? What are you expecting - FFID metadata for all files belonging to the given consignment?

Reply:
I've changed it to `return multiple rows and matches for multiple files` which is a bit more descriptive but I'm open to suggestions.

Reply:
Sorry, I hadn't pushed the change.